src/compiler.c

```diff
@@ -91,11 +91,11 @@ int Compiler_compile(Compiler* compiler, const GherkinDocument* gherkin_document
                 if (!example_table->table_header) {
                     continue;
                 }
-                const PickleTags* tags = create_pickle_tags(feature->tags, scenario_outline->tags, example_table->tags);
                 int l;
                 for (l = 0; l < example_table->table_body->row_count; ++l) {
                     const TableRow* table_row = &example_table->table_body->table_rows[l];
                     const PickleLocations* locations = PickleLocations_new_double(table_row->location.line, table_row->location.column, scenario_outline->location.line, scenario_outline->location.column);
+                    const PickleTags* tags = create_pickle_tags(feature->tags, scenario_outline->tags, example_table->tags);
                     PickleSteps* steps = (PickleSteps*)malloc(sizeof(PickleSteps));
                     steps->step_count = scenario_outline->steps->step_count +  + background_step_count;
```

Comment:
I don't want to be out of order here but is this line correct? There are two additions here.  Should there only be one?

Reply:
From what I can tell @brasmusson moved line 94 to line 98 that's all. One deletion, one addition.

Reply:
@aslakhellesoy it's not that. It is line 100.  There are two pluses there where I think there should only be one:

```
steps->step_count = scenario_outline->steps->step_count +  + background_step_count;
```
-----------------------------------------------------------------------------------------^^^^^

Reply:
Oh that's weird I agree. It was there before this PR though.

Reply:
Indeed. How you would like me proceed? Should I file a bug?

Reply:
I'll remove a `+` and if it passes I'll just commit it :-P